lntest/node/config.go

```diff
@@ -233,7 +232,12 @@ func (cfg *BaseNodeConfig) GenArgs() []string {
 		// Use a small batch delay so we can broadcast the
 		// announcements quickly in the tests.
 		"--gossip.sub-batch-delay=5ms",
+
+		// Use a small cache duration so the `DescribeGraph` can be
+		// updated quicker.
+		"--caches.rpc-graph-cache-duration=100ms",
```

Comment:
Cool, should also make sure it's aligned with the poll interval of the stuff in `wait`.